### DISCUSSION TITLE
Confused by how Ctrl-g works

### DISCUSSION THREAD

**Discussion by xuhdev** [tridactyl/tridactyl#5125] [Q&A] [answered]
If I hit `Ctrl-g`, type a letter that appears on the page, then hit `Esc`, it looks like I can never get the find box back.

Am I missing something? How should I get the find box back?

I really like to retain `Ctrl-f` as scrolling down page and prefer to not unbind it.

**Comment by bovine3dom** [comment, marked as answer]
If you press ctrl-v ctrl-f or ctrl-g ctrl-f you should get the standard Firefox find box

ctrl-g is nothing to do with Tridactyl by the way, it's a Firefox feature :)
